lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -552,7 +551,7 @@ def _partition_pdf_batched_named(
             with LogTime("pdfminer_supplement"):
                 for d, p in zip(deformable_layout, pdfminer_layout):
                     self._supplement_text(d, p)
-        for i in batches:
+        for i in convert_from_path_streamed_batched(filename, batch_size):
```

Comment:
`# TODO: optimize this to make pdfminer also streamed so we can process each page in sequence without having to double-convert the document`